lib/work_bookmark_count_caching.rb

```diff
@@ -0,0 +1,15 @@
+module WorkBookmarkCountCaching
+  def key_for_bookmark_counting_public
```

Comment:
I hate to be picky but I find the naming here confusing. Can I suggest for clarity and consistency: 

BookmarkCountCaching/bookmark_count_caching.rb
public_bookmarks_count
key_for_public_bookmarks_count
invalidate_public_bookmarks_count
/v1/public_bookmarks_count/#{self.id}